src/System.Management.Automation/FormatAndOutput/common/FormatViewGenerator_Table.cs

```diff
@@ -173,7 +173,10 @@ private TableHeaderInfo GenerateTableHeaderInfoFromDataBaseInfo(PSObject so)
                     ci.alignment = colHeader.alignment;
                     if (colHeader.label != null)
                     {
-                        ci.HeaderMatchesProperty = so.Properties[colHeader.label.text] is not null;
+                        if (colHeader.label.text != string.Empty)
+                        {
+                            ci.HeaderMatchesProperty = so.Properties[colHeader.label.text] is not null;
+                        }
```

Comment:
After the change, `ci.HeaderMatchesProperty` is unset in this code path when `label.text == string.empty`, is that expected?

Reply:
There shouldn't be any problem with it being unset as it defaults to `true`